scripts/docker/buildpack-deps/Dockerfile.emscripten

```diff
@@ -39,8 +43,8 @@ RUN set -ex; \
 	mkdir build; \
 	cd build; \
 	emcmake cmake \
+		-DCMAKE_INSTALL_PREFIX=$(em-config CACHE)/sysroot/usr \
```

Comment:
Can you pull this out into a config variable so that the two copies are kept in sync?

Reply:
I probably could... but do you think that warrants rebuilding everything :-)?
It's a bug that it's required here anyways, so I'm pretty sure we'll need to change this on the next update anyways.

Reply:
I actually thought about adding a comment about removing it here, but wasn't sure how nicely that would play with the line break escape, so I added the one on top.

Reply:
Also there's little danger updating this incorrectly - if it's not done right, the builds will fail.

Reply:
I'm not too set on it.